pkgs/development/libraries/libcoro/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv
+, cmake
+, fetchFromGitHub
+, openssl
+, c-ares
+, tl-expected
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libcoro";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "jstranik";
+    repo = "libcoro";
+    rev = "459692cad0bd4814e2bc655f39d406897af07a4a";
+    sha256 = "sha256-tZRCRKG8J9Ngk5x0d1eydziBW32HwmI8GIQRIzckL7Q=";
+    fetchSubmodules = true;
+    # url = "https://github.com/jbaldwin/libcoro.git";
+    # rev = "main";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [ cmake  ];
+
+  propagatedBuildInputs = [ c-ares openssl.dev tl-expected];
```

Please re-format this.
```suggestion
  nativeBuildInputs = [ cmake ];

  propagatedBuildInputs = [ c-ares openssl.dev tl-expected ];
```